extension/lib.js

```diff
@@ -39,6 +43,9 @@ function getFuncsForInjection (usePerformanceFix) {
                 // We manually increase the sequence number.
                 // TODO: Find better solution?
                 textStr = self._lastStreamChunk;
+                if (textStr === undefined) {
```

Comment:
Can textStr or self._lastStreamChunk really be undefined?

Reply:
`self._lastStreamChunk` will currently only get set if playlists are served that contain ads and non-ads. So as long as that scenario isn't fulfilled (e.g. ad-only preroll or midroll playlists), `self._lastStreamChunk` will be undefined.

Reply:
I did a try-catch with console.trace in the catch to diagnose the error accessing [1] in getSeqNr.
It indicated that textStr was undefined there.